hanami-utils.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.0'
 
   spec.add_dependency 'transproc', '~> 1.0'
+  spec.add_dependency 'wisper', '~> 2.0.0'
```

Comment:
We can make `wisper` an optional dependency for `Hanami::Utils`, by only requiring it for `Hanami::Events`, not all of `Hanami::Utils`:
```
if defined?(Wisper)
  require 'hanami/events/backends/wisper' # or something...
else
  raise "You must require `wisper` in your Gemfile in order to use `Hanami::Events`"
end
``` 
This would also let us add different backends later, which could be useful.